site/src/pug/includes/body.pug

```diff
@@ -11,23 +11,25 @@ include ./fork.pug
 
         .buttons.flex
 
-            .item.flex
-                #codefund_ad
```

Comment:
This is needed to cover up our hosting costs. 😅

Reply:
Whoops! Didn't realise I deleted that :sweat_smile:

Reply:
No problem; check out the new layout (bottom bar). Let me know if you think top bar is a more suitable place..

Reply:
The ad goes out-of-screen currently because it is nested in the `.buttons` div. Also, the bottom part of the screen is getting covered by the ad, so some margin will be required.

👍 for bottom bar though!

Reply:
So I should probably move it out to the top-most level, right? Not sure how to add margin though!

Reply:
Should I create a PR? It's a minor fix

Reply:
I think I managed to fix it. Can you check?

Reply:
Oh, I thought you wanted to add margin between banner and site content 😂